### ISSUE TITLE
false positive, unused variables, SYSTEM-CALL

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1699]
from https://github.com/se38/zJSON

```ABAP
DATA c_last_error TYPE i.
DATA ihttp_scid_base64_unescape_x TYPE i VALUE 87.
DATA tmp_s TYPE string.
DATA lv_string TYPE string.

SYSTEM-CALL ict
  DID
    ihttp_scid_base64_escape_x
  PARAMETERS
    tmp_s
    lv_string
    c_last_error.
```
